src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitString = sourceString
```

Comment:
The assignment of `sourceString` to `splitString` is incorrect. The `sourceString` should be directly used in the `split` method without the assignment operator `/`. It should be `const splitString = sourceString.split(';')`.